01 - Community/People/lkadre.md

```diff
@@ -0,0 +1,67 @@
+---
+aliases:
```

Comment:
I'd probably add your other nicks (akhilleus etc.) for findability :)

Reply:
I'll mark this one as solved since this should be optional for authors.